concordium-consensus/src/Concordium/KonsensusV1/Types.hs

```diff
@@ -919,3 +965,129 @@ computeBlockHash bhh bqh =
 
 instance HashableTo BlockHash BakedBlock where
     getHash bb = computeBlockHash (getHash bb) (getHash bb)
+
+-- |A collection of signatures
+-- This is a map from 'FinalizerIndex' to the actual signature message.
+newtype SignatureMessages a = SignatureMessages
+    { qsmFinMessages :: IntMap.IntMap a
+    }
+    deriving (Eq, Show, Generic)
+
+-- |Construct an empty 'SignatureMessages'
+emptySignatureMessages :: SignatureMessages a
+emptySignatureMessages = SignatureMessages IntMap.empty
+
+-- |Serialize instance for 'SignatureMessages QuorumSignatureMessage'
+-- This is just a generic instance for serializing the IntMap which should
+-- be fine as 'SignatureMessages' is not part of the protocol.
+instance Serialize (SignatureMessages QuorumSignatureMessage)
+
+-- |Serialize instance for 'SignatureMessages TimeoutSignatureMessage'
+-- This is just a generic instance for serializing the IntMap which should
+-- be fine as 'SignatureMessages' is not part of the protocol.
+instance Serialize (SignatureMessages TimeoutSignatureMessage)
+
+-- |A 'BlockItem' together with its verification result.
+-- Precondition: The verification result must be 'Ok'.
+-- The verification result serves as a witness which the
+-- scheduler can possibly use to short circuit some verification steps
+-- before executing.
+data VerifiedBlockItem = VerifiedBlockItem
+    { -- |The block item
+      vbItem :: !BlockItem,
+      -- |The associated verification result.
+      vpVerRes :: !TVer.VerificationResult
+    }
```

Comment:
1) The comment seems misleading. It's not always `Ok`. It can be the case that it's `MaybeOk` if the transaction could not be deemed `Ok` (and it was received as part of a block). 

2) But I think this particular type can be categorized as remnants of some early prototyping.

Reply:
Removed it for now.